### PR TITLE
[Fixed] Added .spit and .retarded

### PR DESCRIPTION
**Pull request by StellaraUmbrix**
WHAT CAN I SAY EXCEPT Y O U R W E L C O M E?

### CODE REVIEW

File: src/Client.js [hunk 1]
```diff
@@ -157,7 +157,26 @@ class Client {
...
     return this._get("/alert", { text });
   }
+  
+   /**
+   * S P I T O N H I M
+   * @param {String} firstImage - First image to spit
+   * @param2 {String} secondImage - Second image to spit
+   * @returns {Promise<Buffer>}
+   */
+  spit(firstImage, secondImage) {
```
Comment (ravener) [inline src/Client.js:167]: missing closing curly for this function

**Review by ravener — CHANGES_REQUESTED** [review]
SyntaxError here and missing typings

File: src/Client.js [hunk 2]
```diff
@@ -157,7 +157,26 @@ class Client {
...
+    return this._get("/spit", { firstImage, secondImage });
 
+   /**
+   * You r e t a r d e d?
+   * @param {String} image - What to make retarded
+   * @returns {Promise<Buffer>}
+   */
+  alert(image) {
```
Comment (StellaraUmbrix) [inline src/Client.js]:
```suggestion
  retarded(image) {
```

**Review by StellaraUmbrix — COMMENTED** [review]
Fixed again, finally

**Review by StellaraUmbrix — COMMENTED** [review]
Fixed

**Pull request merged**